### DISCUSSION TITLE
Welcome to Notzz-App Discussions!

### DISCUSSION THREAD

**Discussion by Spikeysanju** [TheCodeMonks/Notzz-App#73] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rodrigolmti** [comment]
Hi just some questions

1. Any reason to use serializable instead of parcelable in your entity model? (Parcel has a better performance than serializable).

2. And I did not found a base behavior for your fragments that you clear your binding instance (this is a common memory leak problem for fragments).

**Reply by Spikeysanju** [reply]
@Rodrigolmti  First of all thanks for bringing this up 🙌
1. I agreed. `Parcelize` has a better performance than `Serializable`. 
2. Yes we need to clear our `binding instances`. However I implemented this for one of my OS project called [NYTimes](https://github.com/TheCodeMonks/NYTimes-App/blob/master/app/src/main/java/www/thecodemonks/techbytes/ui/base/BaseFragment.kt) . 

Are you interested to work on the above issues?. Please LMK & Thanks in advance.
